presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -84,6 +84,14 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Tables with Unsupported Columns
+-------------------------------
+
+If you query a PostgreSQL table with the Presto connector, and the table either has no supported columns or contains
+only unsupported data types, Presto will return an error. The error message will be similar to the following example:
```

Comment:
```suggestion
only unsupported data types, Presto returns an error similar to the following example:
```
Suggestion as shorter and simpler, and to avoid future tense in documentation.